src/LibObjectFile/Dwarf/DwarfHelper.cs

```diff
@@ -31,35 +31,21 @@ public static uint SizeOfUInt(DwarfAddressSize addressSize)
 
         public static uint SizeOfULEB128(ulong value)
         {
-            if (value == 0) return 1;
+            // bits_to_encode = (data != 0) ? 64 - CLZ(x) : 1 = 64 - CLZ(data | 1)
+            // bytes = ceil(bits_to_encode / 7.0);            = (6 + bits_to_encode) / 7
+            uint x = 6 + 64 - (uint)BitOperations.LeadingZeroCount(value | 1UL);
```

Comment:
chi is failing with BitOperations not found, probably missing a using System.Numerics

Reply:
Oops, missed that.. :facepalm: Should be fine now.